code/modules/client/client procs.dm

```diff
@@ -400,9 +400,10 @@
 
 	if(!winexists(src, "asset_cache_browser")) // The client is using a custom skin, tell them.
 		to_chat(src, "<span class='warning'>Unable to access asset cache browser, if you are using a custom skin file, please allow DS to download the updated version, if you are not, then make a bug report. This is not a critical issue but can cause issues with resource downloading, as it is impossible to know when extra resources arrived to you.</span>")
-
-	if(SSinput.initialized)
-		set_macros()
+	
+	if (SSinput.initialized)
+		spawn(30) // without this input won't work on first connect
```

Comment:
This line fixed the issue

Reply:
That's wack, I wonder why it has to chill for so long. Maybe its init order should be bumped down instead? Hm...

Reply:
@variableundefined perhaps we could think of a better way to deal with this? The problem was fixed locally by doing this